docs/getting-started.txt

```diff
@@ -0,0 +1,50 @@
+.. _getting-started:
+
+===============
+Getting Started
+===============
+
+Install and get started with the :ref:`CrateDB Npgsql Plugin <index>`.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Prerequisites
+=============
+
+.. CAUTION::
+
+    For now, this plugin depends on a `fork`_ of the official `Npgsql
+    project`_ to work. We are, however, working to get the necessary changes
+    merged in upstream.
+
+For convenience, we provide a forked `Npgsql installer`_ that you can download
```

Comment:
Mention that the MSI Installer (=GAC installer) is **NOT** the recommended way. It is only necessary for 3rd party apps like Microsoft PowerBI. In some cases you'll want to install Npgsql into your Global Assembly Cache (GAC). This is usually the case when you're using a generic database program that can work with any ADO.NET provider but doesn't come with Npgsql or reference it directly. For these cases, you can download the Npgsql Windows installer from `cnd.crate.io`

Reply:
Check http://www.npgsql.org/doc/index.html#gac-installation